src/index.js

```diff
@@ -845,6 +821,27 @@ Biconomy.prototype.accountLogin = async function(signer, signature, cb) {
 		});
 }
 
+const getLoginTransactionReceipt = async (engine,txHash,userAddress) => {
+    var receipt = await web3.eth.getTransactionReceipt(txHash);
+    if(receipt){
+      	if(receipt.status){
+        	await _getUserContractWallet(engine, userAddress, (error, userContract) => {
+				if(!error && userContract) {
+					_setLocalData(userAddress, userContract);
+					engine.isLogin = true;
+					eventEmitter.emit(EVENTS.LOGIN_CONFIRMATION, "User Contract wallet created Successfully", userContract);
+				}
+			});
+      	}
+      	else if(!receipt.status){
+			eventEmitter.emit(EVENTS.BICONOMY_ERROR, "User Contract wallet creation Failed");
```

Comment:
Send an object in response using formatMessage(code,message) function already present in this file.. Use a error code from config.RESPONSE_CODES object